scripts/style.sh

```diff
@@ -11,16 +11,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-find . \
-    -name 'third_party' -prune -o \
-    -name 'Auth' -prune -o \
-    -name 'AuthSamples' -prune -o \
-    -name 'Database' -prune -o \
-    -name 'FirebaseCommunity.h' -prune -o \
-    -name 'Messaging' -prune -o \
-    -name 'Pods' -prune -o \
-    -path '*/Firestore/Port/*' -prune -o \
-    \( -name '*.[mh]' -o -name '*.mm' -o -name '*.cc' \) \
-    -not -name '*.pbobjc.*' \
-    -not -name '*.pbrpc.*' \
-    -print0 | xargs -0 clang-format -style=file -i
+find . -type f | sed -E -n '
+# Build outputs
+/\/Pods\// d
```

Comment:
Optional nit: Regexs with slashes in them are ugly due to all the escaping. But you can typically use different separator characters to un-uglify it, at least a little.

eg. `/\/Pods\//` might become `\,/Pods/,` (using the comma as the separator. You need to prefix the first one with a backslash to make that work, but the rest don't need it.)